common/const/expansions.ts

```diff
@@ -0,0 +1,27 @@
+type ExpansionInfo = {
+	name: string
+}

```

Comment:
I think its overkill to have properties for the expansion when there is no other properties